app/components/c-add-to-cart.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div :class="b()">
+    <e-input
+      ref="quantity"
+      :class="b('quantity')"
+      v-model="quantity"
```

Comment:
- [ ] Please use typecaster to we can work with Numbers: https://vuejs.org/v2/guide/forms.html#number